atlp-rwanda/eagles-ec-be
# redirect link of the email reset password updating

**What this PR do ?**

PR was using BE URL link to navigate a user to the page for resetting password so we change that URL to the deployed version of the FE that will allow to work well on the front-end as well as expected.

**Description of the task**

This allow user to click on the link sent inbox and navigate him/ her to the page for resetting a password and this will work on the front-end on deployed version.

How to test this task manually ?

1. Clone the repository.
2. Checkout to the branch fix-resetpassword-redirect
3. Run npm install to install dependencies.
4. Run npm run dev to start the development server and login page and click to forget password.
5. Insert your email used for registering.
6. Check inbox if the link being sent to you.
7. Click on that link and go to the page for reset password
8. Enter new password and confirm it 
9. Click to reset and should be able to see notification says you successfully updated your password.
10. Congulaturation you have been updated your password and check confirmation email in your inbox.
